lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
+			"get": {
+				"description": "Gets value from k/v store",
+				"tags": [
+					"kv"
+				],
+				"operationId": "GetValue",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "key",
+						"type": "string",
+						"in": "query",
+						"required": true
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/KeyValue"
+						}
+					},
+					"404": {
+						"description": "Not found"
+					},
+					"default": {
+						"description": "error",
+						"schema": {
+							"$ref": "#/definitions/Error"
+						}
+					}
+				}
+			},
+			"delete": {
+				"description": "deletes entry in k/v store",
+				"tags": [
+					"kv"
+				],
+				"operationId": "DeleteValue",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+					"name": "key",
+					"type": "string",
+					"in": "query",
+					"required": true
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK"
+					},
+					"404": {
+						"description": "Not found"
+					},
+					"default": {
```

Comment:
default is what's specified in the swagger json (generic error), but in the handler I set the HTTP code to 500 -- so I'm returning a 500 if / when it goes sideways